pkg/irreparable/irreparable_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"storj.io/storj/internal/testcontext"
+)
+
+func TestIrreparableRepository(t *testing.T) {
+	//testing variables
+	var (
```

Comment:
You can create a test segment here instead of each variable separately.

Reply:
done